test-data/unit/check-generics.test

```diff
@@ -945,13 +945,17 @@ S = TypeVar('S', int, list)
 class A(Generic[T, S]):
     def __init__(self, x: T, y: S) -> None: ...
 
-BadA = A[str, T] # E: Bad ...
+BadA = A[str, T]  # This error is reported twice (but it actually looks useful)
 SameA = A[T, T]
 
 x = None # type: SameA[int]
-y = None # type: SameA[str] # E: Bad ...
+y = None # type: SameA[str] # E: Invalid type argument value for "A"
 
 [builtins fixtures/list.pyi]
+[out]
+main:8: error: Invalid type argument value for "A"
+main:8: error: Type argument 1 of "A" has incompatible value "str"
```

Comment:
I don't like getting this twice. The first error is too vague since it doesn't tell you which position has the error.

(Unless you turn on column numbers? Which reminds me, it would be nice to turn on `--show-column-numbers` for at least some of the tests, to verify that they're being passed along.)